Packs/rasterize/ReleaseNotes/2_0_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Fixed an issue where **rasterize** did not take into consideration the **width** and **height** parameters.
```

Comment:
```suggestion
Fixed an issue where **rasterize** did not take into consideration the *width* and *height* parameters.
```
Do you mean the rasterize integration?  If so, I would write 
Fixed an issue where the **rasterize** integration did not take into consideration the *width* and *height* parameters.